explore-htap.md

```diff
@@ -31,6 +31,24 @@ The following are the typical use cases of HTAP:
 
 For more information about use cases of TiDB HTAP, see [blogs about HTAP on the PingCAP website](https://en.pingcap.com/blog/?tag=htap).
 
+To enhance the overall performance of the TiDB database, it is recommended to use HTAP in the following technical scenarios:
+
+- Improve analytical processing performance
+
+    If your application involves complex analytical queries, such as aggregation and join operations, and these queries are performed on a large amount of data (more than 10 million rows), the row-based storage engine [TiKV](/tikv-overview.md) might not meet your performance requirements if the tables in these queries cannot effectively use indexes or have low index selectivity.
+
+- Hybrid workload isolation
+
+    In high-concurrency Online Transactional Processing (OLTP) scenarios, you might also have to handle OLAP workloads. To ensure the overall system stability, you need to avoid the impact of OLAP queries on OLTP performance.
```

Comment:
```suggestion
   While dealing with high-concurrency Online Transactional Processing (OLTP) workloads, your system might also need to handle some OLAP workloads. To ensure the overall system stability, you expect to avoid the impact of OLAP queries on OLTP performance.
```